### PR TITLE
Add extra note for open ports

### PR DESCRIPTION
**Pull request by ajgajg1134**
### Additional Notes

---

### Reviewer checklist
- [ ] Review the changed files.
- [ ] Review the URLs listed in the [Preview](#preview) section.
- [ ] Check images for PII
- [ ] Review any mentions of "Contact Datadog support" for internal support documentation.

### CODE REVIEW

**Review by pawelchcki — APPROVED** [review]
LGTM

File: content/en/agent/guide/network.md [hunk 1]
```diff
@@ -204,7 +204,9 @@ All outbound traffic is sent over SSL through TCP / UDP.
 </div>
 
 Open the following ports to benefit from all the **Agent** functionalities:
-
+<div class="alert alert-warning">
+You should ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access could allow malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, or obtaining information about your configuration and services.
+</div>
```
Comment (urseberry) [inline content/en/agent/guide/network.md:209]:
```suggestion
<div class="alert alert-warning">
Ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access allows malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, and obtaining information about your configuration and services.
</div>
```
Also, move this whole section to **before** "Open the following ports..." The current location is interrupts the instruction about which ports to open.

**Review by urseberry — CHANGES_REQUESTED** [review]

File: content/en/agent/guide/network.md [hunk 2]
```diff
@@ -201,10 +201,10 @@ Add all of the `ip-ranges` to your inclusion list. While only a subset are activ
 
 <div class="alert alert-warning">
 All outbound traffic is sent over SSL through TCP / UDP.
+Ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access can allow malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, or obtaining information about your configuration and services.
```
Comment (urseberry) [inline content/en/agent/guide/network.md:204]:
```suggestion
<br>
Ensure the Agent is only accessible by your applications or trusted network sources using a firewall rule or similar network restriction. Untrusted access can allow malicious actors to perform several invasive actions, including but not limited to writing traces and metrics to your Datadog account, or obtaining information about your configuration and services.
```

File: content/en/agent/guide/network.md [hunk 3]
```diff
@@ -201,10 +201,11 @@ Add all of the `ip-ranges` to your inclusion list. While only a subset are activ
 
 <div class="alert alert-warning">
 All outbound traffic is sent over SSL through TCP / UDP.
+<br>
```
Comment (urseberry) [inline content/en/agent/guide/network.md:204]:
```suggestion
<br><br>
```

**Review by urseberry — APPROVED** [review]

**Pull request merged**